components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -3172,6 +3172,68 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  ######################################################
+  # The tenant resource APIs
+  ######################################################
+  '/tenants':
+    #--------------------------------------------------------
+    # Retrieve tenants by state
+    #--------------------------------------------------------
+    get:
+      security:
+        - OAuth2Security: []
+      x-wso2-curl: "curl -X GET \"https://localhost:9443/api/am/store/v1.0/tenants?state=active\" -H \"Authorization: Basic 3a7ecd76-4856-33ce-a703-e23e6cd24944\""
```

Comment:
Let's add examples as mentioned in https://github.com/wso2/carbon-apimgt/wiki/Adding-Examples-for-REST-APIs

Reply:
fixed https://github.com/wso2/carbon-apimgt/pull/6811/commits/a840167e827f482f100d25dadea5b4a46dc53be1